lib/components/circular_button.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:FoodForGood/constants.dart';
+import 'package:flutter/material.dart';
+
+class CircularButton extends StatelessWidget {
+  final Color splashColour;
+  final Color colour;
+  final String title;
+  final Function pressed;
+  final height;
+  final width;
+
+  CircularButton({
+    this.colour,
+    this.title,
+    this.pressed,
+    this.splashColour = Colors.deepOrange,
+    this.height = 150.0,
+    this.width = 150.0
+  });
```

Comment:
Make sure the order is maintained. (`splashColour` should be after `pressed`).

Reply:
Done